pkg/controller/command/command_config_bootstrap.go

```diff
@@ -266,3 +268,48 @@ resources:
       public_url: https://{{ .SwiftProxyAddress }}:{{ .SwiftProxyPort }}
     kind: endpoint
 `))
+
+var migrationConfig = template.Must(template.New("").Parse(`
+database:
+  host: {{ .PostgresAddress }}
+  user: {{ .PostgresUser }}
+  password: {{ .PGPassword }}
+  name: {{ .PostgresDBName }}_migrated
+  max_open_conn: 100
+  connection_retries: 10
+  retry_period: 3s
+  replication_status_timeout: 10s
+  debug: false
+
+log_level: debug`))
+
+var migrationScriptConfig = template.Must(template.New("").Parse(`
+#!/usr/bin/env bash
+
+set -e
+export PGPASSWORD={{ .PGPassword }}
+
+# Try to drop old databases the may or may not exists.
+psql -w -h {{ .PostgresAddress }} -U {{ .PostgresUser }} -d {{ .PostgresDBName }} <<END_OF_SQL
+DROP DATABASE {{ .PostgresDBName }}_migrated;
+DROP DATABASE {{ .PostgresDBName }}_backup;
```

Comment:
[nit] another nit is that there is a command line tool for database deletion `dropdb` so you don't have to write SQLs; https://www.postgresql.org/docs/9.3/app-dropdb.html#:~:text=dropdb%20destroys%20an%20existing%20PostgreSQL,the%20SQL%20command%20DROP%20DATABASE.

Reply:
It has `--if-exists` flag so it does not throw an error when the db does not exist

Reply:
done